code-studio/src/js/components/progress/progress_dot.jsx

```diff
@@ -156,49 +162,73 @@ export const ProgressDot = React.createClass({
     const outlineCurrent = this.props.courseOverviewPage && onCurrent;
     const smallDot = !this.props.courseOverviewPage && !onCurrent;
     const showLevelName = level.kind === 'named_level' && this.props.courseOverviewPage;
+    const isPeerReview = level.kind === 'peer_review';
+    const isLockedReview = isPeerReview && level.status === 'not_tried';
```

Comment:
We had discussed `locked` being a property on the puzzle dot, not something specific to peer review.